docs/repos.rst

```diff
@@ -214,14 +214,19 @@ directory (i.e. local repos), all we have to do is this:
 
 .. code-block:: bash
 
-    $ repobee repos migrate --assignments task-1 task-2
+    $ repobee repos migrate --allow-local-templates --assignments task-1 task-2
 
 .. note::
 
     It may seem a bit odd that the ``--assignments`` option is used to specify
     the repos to migrate. This is an implementation detail that makes it easier
     to handle the command, but may be changed in the future for better
     usability.
+    
+    The --allow-local-templates oprion is necessary to permit the use of a local
+    template on your computer, i.e., the assignment templates that you are migrating.
+    Repobee prohibits using local templates by default for safety, but this is
+    precisely what you want in this case.
```

Comment:
I just have some very minor notes on formatting here, and a typo. I'd also like to change "safety" to "avoid user error", as the former makes it sound like a security feature, which it really isn't.

```suggestion
    The ``--allow-local-templates`` option is necessary to permit the use of a local
    template on your computer, i.e., the assignment templates that you are migrating.
    RepoBee prohibits using local templates by default to avoid user error, but this is
    precisely what you want in this case.
```